webapp/javascript/components/TimelineChart/types.ts

```diff
@@ -0,0 +1,49 @@
+export type PlotType = {
```

Comment:
I wonder if we can't leverage typings from `@types/flot` -> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/flot/index.d.ts#L217